nixos/modules/installer/tools/tools.nix

```diff
@@ -69,16 +69,7 @@ let
     manPage = ./manpages/nixos-version.8;
   };
 
-  nixos-enter = makeProg {
-    name = "nixos-enter";
-    src = ./nixos-enter.sh;
-    inherit (pkgs) runtimeShell;
-    path = makeBinPath [
-      pkgs.util-linuxMinimal
-    ];
-    manPage = ./manpages/nixos-enter.8;
-  };
-
+  nixos-enter = pkgs.nixos-enter;
```

Comment:
nit: could be moved up to the inherit from nixos-option.

Reply:
No longer part of my diff.